movie.sh

```diff
@@ -1,7 +1,7 @@
 #!bin/bash
 function getrating()
 {
-curl "http://www.omdbapi.com/?t=$1&y=&plot=short&r=xml" > mov.txt
```

Comment:
You should indent your code inside the function. Read this: https://google.github.io/styleguide/shell.xml#Indentation